CollAction/Views/Home/FAQ.cshtml

```diff
@@ -8,6 +8,7 @@
 
 @section styles {
     <link rel="stylesheet" href="~/css/faq.css" />
```

Comment:
This is an odd one here.... looks like legacy. Isn't the faq.scss already included by webpack?